_posts/2024-03-19-WildFly-mini-conference-recap.adoc

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "WildFly Mini Conference"
```

Comment:
Maybe we could use "WildFly Mini Conference Recap" as the title? WDYT?

Reply:
TBH, I had "Recap" in the title in the first place, but I wasn’t sure if that was good English. Changing back to include "Recap"